cli/cli_app.go

```diff
@@ -544,6 +546,7 @@ func runTerraformInit(terragruntOptions *options.TerragruntOptions, terragruntCo
 		} else {
 			// Terraform versions >= 0.10.0 specify the module source using the -from-module option
 			initOptions.AppendTerraformCliArgs("-from-module="+terraformSource.CanonicalSourceURL.String(), "-no-color")
+			initOptions.TerraformCommand = CMD_INIT_FROM_MODULE
```

Comment:
Might want to set this in the if-clause above too... Terraform before 0.10.0 did the behavior of `-from-module` without the `-from-module` param...

Reply:
Done